### ISSUE TITLE
Does it work on linux?

### ISSUE THREAD

**Issue by sidvud98** [GSam/Capture2Text#4]
Has the code for installing it in linux been completed?
Please update

**Comment by GSam** [comment]
It works only on the command line, no GUI, and the instructions on the in the README.md. Last time I tried it was on Ubuntu 16.04.

**Comment by sidvud98** [comment]
Is it not completely coded

**Comment by GSam** [comment]
I only ported some parts of the build scripts to Linux, it's not my project. Some obviously components don't work on Linux (or their APIs need some reworking).

**Comment by sidvud98** [comment]
I'm just a beginner to opensource and software development. Can you please elaborate if I can use that command line to take screenshots of selected area and use ocr to extract text from it?

**Comment by GSam** [comment]
I don't think so. You can use some other application to generate the selected area and use the OCR to extract text with this command line. For my use, it was enough for me to do what I wanted.
